xcode/Subconscious/Shared/Components/Detail/DetailStackView.swift

```diff
@@ -271,8 +274,31 @@ struct DetailStackModel: Hashable, ModelProtocol {
                 address: address,
                 color: color
             )
+        case let .selectAppendLinkSearchSuggestion(suggestion):
+            switch suggestion {
+            case let .append(address, target):
+                let fx = Just(
+                    DetailStackAction.requestAppendToEntry(target, "\n\(address.markup)")
+                ).eraseToAnyPublisher()
+                
+                return Update(state: state, fx: fx)
+            }
+        // Consider, do we actually want to open the note we appended to?
+        case let .succeedAppendToEntry(address):
+            return update(
+                state: state,
+                action: .pushDetail(
+                    .editor(
+                        MemoEditorDetailDescription(
+                            address: address
+                        )
+                    )
+                ),
+                environment: environment
```

Comment:
I can see an argument for two UX patterns on appending:

1. open the note, showing the newly appended line, with the cursor focused after the new content
2. append the link and close the metasheet, treating it more like a bookmark and preserving the current context

I am personally leaning towards 2 but undecided.